metals/src/main/scala/scala/meta/internal/metals/Compilations.scala

```diff
@@ -66,6 +66,26 @@ final class Compilations(
     cascadeBatch.cancelCurrentRequest()
   }
 
+  def recompile(): Future[Unit] = {
```

Comment:
Used to fully recompile a workspace.

Reply:
if the `fully` part is more important, then maybe it should be called `compileAll` ?

Reply:
Changed to `recompileAll`